platform.md

```diff
@@ -473,10 +483,72 @@ Usage:
 - For each buildpack in `<group>`, if persistent metadata for that buildpack exists in the analysis metadata, lifecycle MUST write a toml representation of the persistent metadata to `<layers>/<buildpack-id>/store.toml`
 - **If** `<skip-layers>` is `true` the lifecycle MUST NOT perform layer restoration.
 - **Else** the lifecycle MUST perform [layer restoration](#layer-restoration) for any app image layers or cached layers created by any buildpack present in the provided `<group>`.
+- When the provided `<group>` contains image extensions (**[experimental](#experimental-features)**), the lifecycle:
+  - MUST record the digest reference to the provided `<build-image>` in `<analyzed>`
```

Comment:
Should we put the `build-image` key in the example at the bottom?

Reply:
Good catch @jabrown85 - added in 4864b74